styleguide/global/modal.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * This file is for zoolander only. It should not be loaded on www as it is
+ * only intended to mimic Drupal 8's modal behavior.
+ */
+
+(($) => {
+  $(() => {
+    // Set default modal settings.
+    $('.rackspaceModalSource').dialog({
+      autoOpen: false,
+      draggable: false,
+      maxHeight: '95%',
+      modal: true,
+      resizable: false,
+      width: '95%',
+    });
+
+    // Classes to add to modal.
+    const classes = {
+      'ui-dialog': 'rackspaceModal',
+      'ui-dialog-title': 'rackspaceModal-title',
+      'ui-dialog-titlebar': 'rackspaceModal-titleBar',
+      'ui-dialog-titlebar-close': 'rackspaceModal-titleBarClose',
+    };
+
+    // Sets proper window position.
+    const positionWindow = () => {
+      const windowWidth = $(window).width();
+      const windowHeight = $(window).height();
+      const modalWidth = windowWidth * 0.95;
+      const modalMaxHeight = windowHeight * 0.8;
+      $('.rackspaceModalSource')
```

Comment:
This is mostly good but I'd recommend caching this lookup since you're using it twice. 
```const modal = $('.rackspaceModalSource');```

Reply:
@runrog changed